### ISSUE TITLE
Some things to review

### ISSUE THREAD

**Issue by rmed** [guluc3m/gul-gultalks#23]
- [ ] Remove nesting from comments (show small quote from parent comment)
- [ ] Show language selector in header

**Comment by miguxbe** [comment]
Added Travis CI.

**Comment by miguxbe** [comment]
Add tests :D

**Comment by yagop** [comment]
How about user signup and login? To edit his own events

**Comment by miguxbe** [comment]
"No users" This thing was a very early requirement in the project hence there are no plans for implement this. But, OTOH this is a thing that I have been thinking for long time and the conclusion reached is: a unique key delivered to the speaker's email could solve the issue because this key should be submitted in order to edit the event. Any comments/thoughts about that?

**Comment by yagop** [comment]
Who the hell wrote that requirement? A gardener? A system administrator? :laughing:
Welcome to the XXI century, were people mistakes and need to edit things. This is something formal, not a warez forum where people regrets signing up.

A random generated token (unique key) would be nice, the token should be associated to an email. But this is a **user** and a **password** for it, isn't it?
The random token (unique key/password) could be delivered to the speaker's email once he submit the event.

**Comment by yagop** [comment]
Add: Notifications when someone writes a comment on yours event.

**Comment by yagop** [comment]
Add: Markdown or any markup language for Description and/or comments. Check [this](https://github.com/vmg/redcarpet) out.

**Comment by miguxbe** [comment]
Yes, maybe a sysadmin... To be precise was @queltos, you have to argue this issue with him :). You have to keep in mind that the app is used a couple times per year more or less, and store users in the DB would be useless and maybe risky. The token is a definitely easier, for the speaker and for us, because we don't have a profile itself with a username, password, other info, etc just a token associated to your email, but of course I know that is nearly the same. And of course the the random key is delivered after submit an event or after click the accept link in the email.

**Comment by yagop** [comment]
Store users on a database is useless and risky? @guluc3m/owners are storing mail lists passwords on plain text and nobody gives a fuck... But this is not the topic now.

I am agree with token based authentication and it would be nice to use the same token for the API to make authenticated requests.

**Comment by rmed** [comment]
I'm not too keen on the idea of having users (apart from admins or assistants), just saying. Also, there is no implementation for performing authenticated requests in place.

**Comment by yagop** [comment]
Improve email templates and fix `<a href="...` code

![http://i.imgur.com/Z5g5zNI.png](http://i.imgur.com/Z5g5zNI.png)
